src/sandstorm/sandstorm-http-bridge.c++

```diff
@@ -2173,6 +2192,35 @@ public:
     });
   }
 
+  kj::Promise<void> connectLoop(kj::Own<kj::NetworkAddress>&& address,
+                                kj::Timer& timer,
+                                bool loggedSlowStartupMessage,
+                                int numTriesSoFar) {
+    return address->connect().then([this](auto x) -> void {
+      KJ_LOG(WARNING, "App successfully started listening for TCP connections!");
```

Comment:
This log message seems overly chatty -- won't it be printed at every startup? IMO better to log only in the problem case.